channeldb/migration_01_to_11/channel.go

```diff
@@ -44,11 +44,57 @@ const (
 	// NOTE: iota isn't used here for this enum needs to be stable
 	// long-term as it will be persisted to the database.
 
-	// SingleFunder represents a channel wherein one party solely funds the
-	// entire capacity of the channel.
-	SingleFunder ChannelType = 0
+	// SingleFunderBit represents a channel wherein one party solely funds
+	// the entire capacity of the channel.
+	SingleFunderBit ChannelType = 0
```

Comment:
Hmm so we're going _back_ to a previously frozen migration an adding new fields? If these are to be used in an upcoming migration, then can we just add the fields there? 

Still counting down to the days where we never need to touch any of this migration stuff again...

Even now I find myself asking if this migration (still getting into the next commits) can be avoided.

Reply:
> Still counting down to the days where we never need to touch any of this migration stuff again...

Counting too...

Reply:
Also removed the adding new fields.